src/test/java/com/scaleoutdata/spring/cloud/stream/kafka/streams/join_example/controller/TestJoinController.java

```diff
@@ -83,10 +81,10 @@ private TopologyTestDriver createTopologyDriver(SpecificAvroSerde specificAvroSe
         final StreamsBuilder builder = new StreamsBuilder();
         KStream<String, Page> inputPage = builder.stream(INPUT_PAGE, Consumed.with(Serdes.String(), specificAvroSerde));
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.